concent_api/verifier/utils.py

```diff
@@ -55,8 +56,23 @@ def store_file_from_response_in_chunks(response: requests.Response, file_path: s
             f.write(chunk)
 
 
-def run_blender():
-    pass
+def run_blender(scene_file, output_format, script_file=''):
+    return subprocess.run(
```

Comment:
It might be good to have few asserts for output_format and scene_file here e.g. output_format is in list of available formats or is scene_file ends with .blend